lib/diagnostic.rb

```diff
@@ -108,10 +110,11 @@
 # Use shorthand to make this one line long.
 # Write your response as a Ruby string.
 
-Diagnostic.person_hash = 'your answer here'
+Diagnostic.person_hash = 'person = { favorite_number: 10, first_name: "nari" }'
 
 # Set a default return value to your `person` hash that concatenates the value
 # of person[:first_name] and a message about them.
 # Write your response as a Ruby string.
 
-Diagnostic.hash_default = 'your answer here'
+Diagnostic.hash_default = 'person = Hash.new { |hash, key| hash[key] = "Sorry, #{key}
+is not willing to share that information with you." }'
```

Comment:
This is good for using the constructor but we were trying to go for the short hand syntax like so:

`'person.default = "#{person[:first_name]} is not willing to share that information with you."'`